python/src/main/java/com/regnosys/rosetta/generator/python/PythonCodeGenerator.java

```diff
@@ -0,0 +1,98 @@
+package com.regnosys.rosetta.generator.python;
+
+import java.util.Collection;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+import com.google.inject.Inject;
+import com.regnosys.rosetta.generator.external.AbstractExternalGenerator;
+import com.regnosys.rosetta.generator.java.RosettaJavaPackages;
+import com.regnosys.rosetta.generator.python.enums.PythonEnumGenerator;
+import com.regnosys.rosetta.generator.python.object.PythonModelObjectGenerator;
+import com.regnosys.rosetta.generator.python.func.PythonFunctionGenerator;
+import com.regnosys.rosetta.rosetta.RosettaEnumeration;
+import com.regnosys.rosetta.rosetta.RosettaMetaType;
+import com.regnosys.rosetta.rosetta.RosettaModel;
+import com.regnosys.rosetta.rosetta.RosettaRootElement;
+import com.regnosys.rosetta.rosetta.simple.Data;
+import com.regnosys.rosetta.rosetta.simple.Function;
+import com.regnosys.rosetta.generator.python.util.PythonModelGeneratorUtil;
+
+
+public class PythonCodeGenerator extends AbstractExternalGenerator {
+	
+	@Inject
+	PythonModelObjectGenerator pojoGenerator;
+	@Inject
+	PythonFunctionGenerator funcGenerator;
+	@Inject
+	private PythonEnumGenerator enumGenerator;
+	@Inject
+	PythonModelGeneratorUtil utils;
+
+	public PythonCodeGenerator() {
+		super("Python");
+		enumGenerator = new PythonEnumGenerator();

```

Comment:
Hi @SimonCockx , thank you for your time.

This PythonGenerator is able to generate enums, objects with conditions/expressions, and methods, but it will only create the definition of the method with the parameters it will use, its content is defined as a pass. There are some static methods that have been defined manually, and the generator, when converting to Python, finds one of those methods, it will get its content. Also, utils.py has defined some utilities to be able to handle metadata.

This is the first milestone and in the future, we will finish the function generator and improve any other aspect we consider. If you have any doubts or comments, please, let me know.

Thanks

Reply:
@dcarrascosa00 Thanks, that's good to know! I'll leave comments in this PR as I go through the code.